src/robot_config.cpp

```diff
@@ -131,6 +133,96 @@ void FetchRobotConfig::joints_for_group(
   }
 }
 int FetchRobotConfig::num_arms() const { return 1; }
+
+BaxterRobotConfig::BaxterRobotConfig() {}
+
+std::string BaxterRobotConfig::planning_frame() const { return "base"; }
+std::string BaxterRobotConfig::planning_group() const { return "both_arms"; }
+std::string BaxterRobotConfig::base_link() const { return "base"; }
+std::string BaxterRobotConfig::torso_link() const { return "torso"; }
+std::string BaxterRobotConfig::joint_states_topic() const { return "/robot/joint_states"; }
+std::string BaxterRobotConfig::ee_frame_for_group(
+    const std::string& actuator_group) const {
+  if (actuator_group == Action::LEFT_ARM) {
+    return "left_gripper";
+  } else if (actuator_group == Action::RIGHT_ARM) {
+    return "right_gripper";
+  } else {
+    return "";
+  }
+}
+
+void BaxterRobotConfig::gripper_joints_for_group(
+    const std::string& actuator_group,
+    std::vector<std::string>* joint_names) const {
+  joint_names->clear();
+  if (actuator_group == Action::LEFT_GRIPPER) {
+    joint_names->push_back("l_gripper_l_finger_joint");
+    joint_names->push_back("l_gripper_r_finger_joint");
+  } else if (actuator_group == Action::RIGHT_GRIPPER) {
+    joint_names->push_back("r_gripper_l_finger_joint");
+    joint_names->push_back("r_gripper_r_finger_joint");
+  }
+}
+void BaxterRobotConfig::gripper_open_positions(
+    std::vector<double>* positions) const {
+  positions->clear();
+  positions->push_back(0.0);
+  positions->push_back(-0.020833);
+  // right
+  // positions->push_back(0.020833);
+  // positions->push_back(0.0);
+
+}
+void BaxterRobotConfig::gripper_close_positions(
+    std::vector<double>* positions) const {
+  positions->clear();
+  positions->push_back(0.0);
+  positions->push_back(0.0);
+  // , 
+}
+void BaxterRobotConfig::default_gripper_poses(
```

Comment:
Is this unused? If so, delete it.

Reply:
Ah sorry, it was used for my condition learning